### ISSUE TITLE
Direct ISO download

### ISSUE THREAD

**Issue by InfRandomness** [FedoraQt/MediaWriter#264]
Hi, 
I'm coming from Etcher which has that option where it can download the ISO right off the internet and write it directly on the USB, I love that feature and I found out that it was missing on fedora media writer, is it possible to add this feature ? it's so handy because we don't have to download the ISO image first and then select it as a Custom Image.

Thanks

**Comment by dreua** [comment]
I think that feature is actually available in MediaWriter, at least for the supported Fedora versions and spins.

**Comment by dreua** [comment]
If there is a USB plugged in you even have the option to start writing automatically when the download is done, I think that's great!

![Screencast from 17 08 2020 23 19 24](https://user-images.githubusercontent.com/15114715/90446612-f459a300-e0e1-11ea-9f94-b46489be06cb.gif)

**Comment by InfRandomness** [comment]
Yes, but that's the case only if you wanna write a fedora edition / spin, not if you wanna write debian or any other kind of distribution for instance

**Comment by dreua** [comment]
Okay, I think this is an important distinction, please put that information in the top post and the issue title to clarify it to other readers. (e.g. "Direct ISO download of 3rd party ISOs")

I'm not a developer here but I assume that the focus of "Fedora Media Writer" is on writing Fedora images specifically. If I'm not mistaken there is also a checksum test for the downloaded images which simply can not be implement for *any* ISO source. In this cases the user should check the integrity of the file to prevent errors and for security before it is written to USB. Therefore in my personal opinion such a feature is not necessarily a good idea.

**Comment by InfRandomness** [comment]
Thanks for the advice, I have modified my post to include that information

**Comment by gridhead** [comment]
Since this is a request on third party ISO images, I think this is not a priority here.

This can be closed.
